pkgs/os-specific/linux/powertop/default.nix

```diff
@@ -32,7 +33,7 @@ stdenv.mkDerivation rec {
 
   postPatch = ''
     substituteInPlace src/main.cpp --replace "/sbin/modprobe" "modprobe"
-    substituteInPlace src/calibrate/calibrate.cpp --replace "/usr/bin/xset" "xset"
+    substituteInPlace src/calibrate/calibrate.cpp --replace "/usr/bin/xset" "${lib.makeBinPath [ xorg.xset ]}/xset"
```

Comment:
why not just do `${pkgs.xorg.xset}/bin/xset` ?

Reply:
I found that and thought the use would be somehow encouraged but I can change for sure ✌🏻️ maybe in case of refactoring something on paths?

Reply:
`lib.getExe xorg.xset` would be more appropriate

Reply:
I don't think refactoring makes sense here, if we need a "local bin path" with more programs, it would be different.

Reply:
I've updated to your solution @RaitoBezarius but pkgs does not make sense here so i used org.xset in the same way? ✌🏻

Reply:
You're right, thank you

Reply:
Though `lib.getExe xorg.xset` is better like @fortuneteller2k said

Reply:
You mean in this case or still only if multiple binaries? Not sure if I should change for PR again? :D

Reply:
`lib.getExe pkg` will automatically give you `pkg/bin/executable-name` if the pkg is well-packaged.
It's better than doing `${xorg.xset}/bin/xset` manually, you can just replace it altogether: `${lib.getExe xorg.xset}`

Reply:
(Please yes change the PR for that)

Reply:
Done ✅